### ISSUE TITLE
denied requests are still showing up in subscription

### ISSUE THREAD

**Issue by leprekon91** [copleykj/socialize-friendships#32]
```
Meteor.publish('friendRequests', function(options){
    if(!this.userId){
        return this.ready();
    }

    options = options || {};

    check(options, publicationOptionsSchema);

    Meteor.publishWithRelations({
        handle: this,
        collection: Meteor.requests,
        filter: {userId:this.userId, denied:{$exists:false}, ignored:{$exists:false}},// Shouldn't this be deniedAt?
        options:options,
        mappings: [{
            key: 'requesterId',
            collection: Meteor.users,
            options:{fields:{username:true, avatarId:true}}
        }]
    });

});
```

In the publication, the filter is applied with `denied: { $exists: false }` but the schema states that it is supposed to be `deniedAt: { $exists: false }`.
This causes the denied requests to still be published to the client. 
Shouldn't it publish only the pending ones? This is also the case in the user extensions and by proxy, the `socialize.friendRequests` publication.

**Comment by StorytellerCZ** [comment]
Ah! Good catch, will submit PR right away!

**Comment by StorytellerCZ** [comment]
In the meantime you can fix it by adding proper filter on client side.

**Comment by StorytellerCZ** [comment]
OK, here is the fix #33
